os/arch/arm/src/amebasmart/amebasmart_spi.c

```diff
@@ -674,6 +681,12 @@ int amebasmart_spi0cmddata(FAR struct spi_dev_s *dev, uint32_t devid, bool cmd)
  ************************************************************************************/
 void amebasmart_spi1select(FAR struct spi_dev_s *dev, enum spi_dev_e devid, bool selected)
 {
+	FAR struct amebasmart_spidev_s *priv = (FAR struct amebasmart_spidev_s *)dev;
+	if (selected == 1) {
+		gpio_write(&priv->gpio_cs0, 0);
+	} else {
+		gpio_write(&priv->gpio_cs0, 1);
+	}
```

Comment:
This is the same as spi0select. Do we need separate functions for them?

Reply:
Yes, no need to separate them. I have updated the PR.